build/charts/flow-aggregator/values.yaml

```
@@ -45,8 +45,25 @@ flowCollector:
 clickHouse:
   # -- Determine whether to enable exporting flow records to ClickHouse.
   enable: false
-  # -- DatabaseURL is the url to the database. TCP protocol is required.
+  # -- DatabaseURL is the url to the database. Provide the database URL as a string with format
+  # <Protocol>://<ClickHouse server FQDN or IP>:<ClickHouse port>. The protocol has to be one of
+  # the following: "tcp", "tls", "http", "https". When "tls" or "https" is used, tls will be enabled.
   databaseURL: "tcp://clickhouse-clickhouse.flow-visibility.svc:9000"
+  # -- tls contains tls related configuration options, which is used to connect to clickhouse service.
```

```suggestion
  # TLS configuration options, when using TLS to connect to the ClickHouse service.
```

you shouldn't make this a documentation comment, otherwise the generated README has duplicate information

---

Fixed. Thanks.